rollout.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+// Rollout is an interface implemented by a value that can provide a rollout
+// percentage as a decimal in the range [0.0,1.0]
+type Rollout interface {
+	// ReadRollout provides the current rollout value.
+	// Note that it is possible for the value to be outside of the
+	// acceptable [0.0,1.0] range - this should be checked by the caller.
+	ReadRollout() float64
```

Comment:
I'm going to call it `Get`, which is concise and doesn't have the conflict with the classic `Read` signature.